uamqp/session.py

```diff
@@ -321,3 +326,30 @@ def create_request_response_link_pair(self, endpoint, **kwargs):
             network_trace=kwargs.pop('network_trace', self.network_trace),
             **kwargs)
 
+    def mgmt_request(self, message, operation=None, operation_type=None, node='$management', **kwargs):
```

Comment:
I would rather not have this function.
I think Management link opening should be handled the same way link opening in handled, then requests should be passed straight  to the management link in the same was that outgoing messages are passed to the link.
We could put logic like this into the clients though. As they are a higher-level implementation.